src/config/constants/pools.ts

```diff
@@ -25,7 +25,7 @@ const pools: PoolConfig[] = [
   {
     // Fixed APR mainnet contract test
     isMain: true,
-    comingSoon: true,
+    comingSoon: false,
```

Comment:
since naka optional naman to sa types, okay lang din ata na wala to. LGTM rin naman siya.